### DISCUSSION TITLE
cyber support qnx

### DISCUSSION THREAD

**Discussion by daohu527** [ApolloAuto/apollo#15563] [Ideas]
We received a proposal from cyber about qnx. The reason for opening this discussion is to clarify some details, such as which version of Apollo it is based on, design documents

**Comment by qinxl1** [comment]
Hello @daohu527 ，

We are ZF AD teams.
Current cyber version we port and support(https://github.com/xuming7up/cyberQNX):
HW: Drive AGX Orin aarch64/Jetson Orin/PC x86_64 + Nvidia RTX4000
OS: Linux ubuntu 20.04(PC+Jetson)/QNX710(Drive AGX Orin)
Cyber: Apollo 6.0 

Can we get a empty branch, we can contribute our code in this branch?
Or could you provide better solution for the contribute?

**Reply by xuming7up** [reply]
@daohu527 
My colleague(Qin) will help to discuss the topic. Thanks!

**Reply by daohu527** [reply]
I think if possible we can organize an online meeting so that we can understand the plan more quickly. You can leave an email address.

I will contact you to confirm the time

**Reply by qinxl1** [reply]
Hello  @daohu527,
Thanks for your reply,my mail: xiaolei.qin@zf.com.
After you book the online meeting(teams is better) I'll forward to the other colleagues also.

**Reply by qinxl1** [reply]
Hello @daohu527 ,
I haven't received your message yet, when can we start our online meeting?

**Reply by daohu527** [reply]
Email has been sent, please check!

**Reply by qinxl1** [reply]
I've replied the email, please check whether the schedule is suitable for you.
